test/sdk/websocket-watch/ws-watch-test.go

```diff
@@ -15,6 +15,7 @@
 package main
```

Comment:
1. setLabel API
When I tested this code of the old version, the test was not passed.

In this test, the setLabel API was called, however, the test mode of LocalSDKServer always reject unless specified label is.
```
{"message":"expected to receive '1655898932' as value for 'setlabel' request but received 'true'","sdkName":"restapi","severity":"error","source":"*sdkserver.LocalSDKServer","time":"2022-06-22T20:55:54.507127+09:00"}
```
https://github.com/googleforgames/agones/blob/main/pkg/sdkserver/localsdk.go#L197-L213

I changed to use setLabel API to reserved API.

Reply:
2. go routine

This test used go routine, however, test result often changed because there is no block with paralell processing.

```
+ cd /go/src/agones.dev/agones/test/sdk/websocket-watch
+ go run ws-watch-test.go
2022/06/22 12:28:09 Connecting to ws://localhost:9101/watch/gameserver
{"message":"Connected to watch GameServer...","sdkName":"restapi","severity":"info","source":"*sdkserver.LocalSDKServer","time":"2022-06-22T12:28:09.1496994Z"}
2022/06/22 12:28:09 Received message from websocket: {"result":{"object_meta":{"name":"local","namespace":"default","uid":"3133294039362009907","resource_version":"v1","generation":"1","creation_timestamp":"1655900878","annotations":{"agones.dev/sdk-UID":"3133294039362009907","annotation":"true"},"labels":{"agones.dev/sdk-creationTimestamp":"1655900878","islocal":"true"}},"spec":{"health":{"period_seconds":3,"failure_threshold":5,"initial_delay_seconds":10}},"status":{"state":"Shutdown","address":"127.0.0.1","ports":[{"name":"default","port":7777}],"players":{"capacity":"10"}}}}
2022/06/22 12:28:09 Could not find label 'agones.dev/sdk-testws' in message
time="2022-06-22T12:28:09Z" level=warning msg="error reading websocket message: websocket: close 1006 (abnormal closure): unexpected EOF"
exit status 1
make[2]: *** [run-sdk-command] Error 1
```